src/store/users/api.ts

```diff
@@ -0,0 +1,9 @@
+import { apiService } from 'services';
+
+import { IDataApi } from './types';
+
+type IGetUsersApi = () => Promise<IDataApi>;
```

Comment:
should be a generic type. you will not write type for each api call. would be better to create a type with general body and use generic type for 'data' property